cubedash/_stac.py

```diff
@@ -165,14 +165,14 @@ def url_for(*args, **kwargs):
 # Conversions
 
 
-def _band_to_measurement(band: dict, dataset_location: str | None) -> MeasurementDoc:
+def _band_to_measurement(band: Mapping[str, Any]) -> MeasurementDoc:
     """Create EO3 measurement from an EO1 band dict"""
     return MeasurementDoc(
-        path=band.get("path"),
+        path=band.get("path", "Unknown"),
         band=band.get("band"),
         layer=band.get("layer"),
-        name=band.get("name"),
-        alias=band.get("label"),
+        name=band.get("name"),  # type: ignore[arg-type]
+        alias=band.get("label"),  # type: ignore[arg-type]
```

Comment:
If I understand eodatasets3 right, these ignores should be possible to remove when https://github.com/opendatacube/eo-datasets/pull/463 is released.